Added Spanish translations of email templates (#806)
carpentries/docs.carpentries.org: topic_folders/instructor_training/email_templates_trainers.md

Quiero que sepas que puedo asignarte cualquiera de los segmentos de la lección que preparaste para enseñar; por lo tanto, prepárate para enseñar todas las partes de la lección que elegiste. Algunas personas se preparan para enseñar solo cinco minutos de una sección en particular y, con frecuencia, deben reprogramar la demostración porque casi nunca les asignan la sección para la que se prepararon.

Una lección corresponde a una sola fila de las tablas de lecciones (https://software-carpentry.org/lessons/, http://www.datacarpentry.org/lessons/ y https://librarycarpentry.org/lessons/) y a un solo repositorio en GitHub. Algunas lecciones tienen módulos complementarios, pero no es necesario que te prepares para enseñar esos módulos en tu demostración de enseñanza.

Por ejemplo, si elegiste “terminal de Unix”, te puedo asignar cualquier sección que figure en http://swcarpentry.github.io/shell-novice/.

> Recuerda que esto no es una prueba determinante, sino una oportunidad favorable para dar y recibir comentarios sobre un estilo de presentación más pulido. Si te pido que repitas tu demostración, te enviaré por correo electrónico información con instrucciones claras sobre lo que se debe cambiar después de la demostración. Durante la demostración, recibirás públicamente solo comentarios cualitativos, no se te informará si este paso de finalización se considera completado.

Reviewer: ```suggestion
Recuerda que esto no es una prueba determinante, sino una oportunidad más para dar y recibir feedback y comentarios sobre cómo enseñar y seguir mejorando. Si te pido que repitas tu demostración, te enviaré por correo electrónico mis comentarios con instrucciones claras para que puedas trabajar y prepararte para la próxima demostración. Durante la sesión, recibirás comentarios y feedback pero no sabrás si aprobaste este paso o no hasta recibir mi mail.
```

Espero verte pronto.

[Nombre de remitente]

#### Invitación para aprendices para que repitan la demostración de enseñanza

> **Nota para las personas que capacitan:** Incluso con una plantilla, puede ser difícil redactar y enviar estos correos electrónicos. Si no están seguras, no duden en comunicarse con la comunidad de personas que capacitan o con el Equipo central para que las ayuden a decidir, personalizar o responder preguntas sobre estos mensajes.

Reviewer: ```suggestion
**Nota para las personas que capacitan:** Incluso con una plantilla, puede ser difícil redactar y enviar estos correos electrónicos. Si tienes dudas, no dudes en comunicarte con la comunidad Trainers o con el Core Team para que te ayuden a decidir, personalizar o responder preguntas sobre estos mensajes.
```

**Asunto:** Capacitación de Instructoras de The Carpentries: demostración de enseñanza

Hola, [nombre de aprendiz]:

> Gracias por hacer una demostración de enseñanza. Aunque demostraste [mencionar algo que haya hecho bien, p. ej., “buen dominio del material de referencia”], te invito a que vuelvas a hacer la demonstración de enseñanza. El motivo principal es que [motivo]. [Explicación de lo que te gustaría ver esta vez y por qué es importante para las prácticas de enseñanza de The Carpentries].

Reviewer: ```suggestion
Gracias por haber participado en la sesión. Aunque demostraste [mencionar algo que haya hecho bien, p. ej., “buen dominio del material de referencia”], te invito a que vuelvas a hacer la demonstración de enseñanza. El motivo principal es que [motivo]. [Explicación de lo que te gustaría ver esta vez y por qué es importante para las prácticas de enseñanza de The Carpentries].
```

> Nos complace tenerte como instructora de The Carpentries y queremos que te sumes a nuestro equipo. Sé que es complicado destinar tiempo para estas sesiones, pero espero que la oportunidad de seguir practicando y de recibir más comentarios sea beneficiosa para ti.

Reviewer: ```suggestion
```
The paragraph says that we are happy to welcome the person as an Instructor. I believe this is a mistake also present in the English version.

Reviewer 2: This may be one of those things that does not translate well. In the English version, the purpose is to reassure the trainee that we DO want them to become an Instructor, even though we are asking them to repeat their demo. Maybe this reassurance could be provided in a different way; otherwise this could be removed for now.

> Me comuniqué con el Equipo central para avisar que esta sesión debe repetirse. Si estás por terminar el período de finalización y necesitas una prórroga, puedes enviar un correo electrónico a [instructor.training@carpentries.org](mailto:instructor.training@carpentries.org) para solicitarla.

Reviewer: ```suggestion
```
Same as before.

Reviewer 2: I don't see the same problem here -- the purpose of this section is to let them know that the Trainer has indicated to the Core Team that the demo needs to be repeated, and they should contact us if this creates problems for them.

Si tienes alguna pregunta, comunícate con nosotras.

Mis mejores deseos.

[Nombre de remitente]

#### Aprendiz aprobó la demostración de enseñanza

**Asunto:** Capacitación de Instructoras de The Carpentries: demostración de enseñanza

Hola, [nombre de aprendiz]:

Me complace informarte que aprobaste la demostración de enseñanza. Demostraste [un buen dominio del material de referencia y una comprensión sólida de los métodos de enseñanza de The Carpentries]. Es un placer tenerte como instructora de The Carpentries.

> Envié esta información a nuestro Equipo central. Si este era el último paso de la finalización de tu capacitación de instructoras, el Equipo central se comunicará contigo en el plazo de una semana aproximadamente.

Reviewer: ```suggestion
Envié esta información a nuestro Equipo central. Si este era el último paso de la certificación para ser instructora, el Equipo central se comunicará contigo en el plazo de una semana aproximadamente.
```
